machine/meta-rcar-gen3/recipes-graphics/gles-module/gles-user-module.bbappend

```diff
@@ -0,0 +1,87 @@
+PVRUM_URL = "git://git@git.epam.com/epmd-aepr/pvr_um_vgpu_img.git"
+BRANCH = "master"
+SRCREV = "${AUTOREV}"
+COMPILER_URL = "${TOPDIR}/../proprietary/rcar/meta/META_Embedded_Toolkit_2.8.1.0.3.1.zip"
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRC_URI_r8a7795 = "${PVRUM_URL};protocol=ssh;branch=${BRANCH}"
+SRC_URI_r8a7796 = "${PVRUM_URL};protocol=ssh;branch=${BRANCH}"
+SRC_URI_remove = " \
+    file://change-shell.patch \
+    file://0001-EGL-eglext.h-Include-eglmesaext.h-to-avoid-compile-error.patch \
+"
+SRC_URI_append = "file://${COMPILER_URL} \
+                  file://pkgconfig/egl.pc \
+                  file://pkgconfig/glesv2.pc \
+"
+
+S = "${WORKDIR}/git"
+PVRUM_BUILD_DIR_r8a7795 = "r8a7795_linux"
+PVRUM_BUILD_DIR_r8a7796 = "r8a7796_linux"
+PVRUM_DISCIMAGE = "${D}"
+BUILD = "release"
+
+DEPENDS += "virtual/kernel wayland-kms libgbm mesa"
+
+EXTRA_OEMAKE += "CROSS_COMPILE=${TARGET_PREFIX}"
+EXTRA_OEMAKE += "PVR_BUILD_DIR=${PVRUM_BUILD_DIR}"
+EXTRA_OEMAKE += "DISCIMAGE=${PVRUM_DISCIMAGE}"
+EXTRA_OEMAKE += "KERNELDIR=${STAGING_KERNEL_BUILDDIR}"
+EXTRA_OEMAKE += "METAG_INST_ROOT=${S}/metag/2.8"
+
+# Need to actually delete the varflag not just unset it.
+python __anonymous() {
+    d.delVarFlag('do_compile', 'noexec')
+}
+
+# Since we depends on wayland-kms we have to provide
+# following header files to make it buildable.
+do_populate_headers() {
+    # Install header files
+    install -d ${STAGING_DIR_TARGET}/${includedir}/EGL
+    install -m 644 ${S}/include/khronos/EGL/*.h ${STAGING_DIR_TARGET}/${includedir}/EGL/
+    install -d ${STAGING_DIR_TARGET}/${includedir}/GLES2
+    install -m 644 ${S}/include/khronos/GLES2/*.h ${STAGING_DIR_TARGET}/${includedir}/GLES2/
+    install -d ${STAGING_DIR_TARGET}/${includedir}/GLES3
+    install -m 644 ${S}/include/khronos/GLES3/*.h ${STAGING_DIR_TARGET}/${includedir}/GLES3/
+    install -d ${STAGING_DIR_TARGET}/${includedir}/KHR
+    install -m 644 ${S}/include/khronos/KHR/khrplatform.h ${STAGING_DIR_TARGET}/${includedir}/KHR/khrplatform.h
+    install -m 644 ${S}/include/khronos/drv/EGL/eglext_REL.h ${STAGING_DIR_TARGET}/${includedir}/EGL/
+}
+
+addtask populate_headers after do_unpack before do_patch
+
+do_configure_prepend() {
+    if [ -f ${WORKDIR}/Meta_Embedded_Toolkit-2.8.1.CentOS-5.tar.gz ]
+    then
+        tar -xzf ${WORKDIR}/Meta_Embedded_Toolkit-2.8.1.CentOS-5.tar.gz
+        rm -f ${WORKDIR}/Meta_Embedded_Toolkit-2.8.1.CentOS-5.tar.gz
+        rm -f ${WORKDIR}/*.pdf
+        ./Meta_Embedded_Toolkit-2.8.1.CentOS-5/install.sh ${S}
+        rm -rf ${S}/Meta_Embedded_Toolkit-2.8.1.CentOS-5
+    fi
+}
+
+do_compile() {
```

Comment:
either put a comment that we override original recipe do_compile in this way or (if not) can it be empty (oe_make is the default do_compile implementation?)